slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -285,7 +316,7 @@ std::string SLAMServiceImpl::GetNextDataFile() {
 
 void SLAMServiceImpl::CreateMap() {
     cartographer::mapping::TrajectoryBuilderInterface *trajectory_builder;
-    int trajectory_id;
+    int trajectory_id = 0;
```

Comment:
[nit] I think remove the initialization with a value of 0? My guess is that this way we'll have to overwrite it somewhere with a meaningful value - else it'll throw an error when we try to use it? 

Don't want `trajectory_id` to be set to a value that's wrong (e.g. a value of `0`) and use that accidentally. Not sure about this, curious what you think @tessavitabile .

Reply:
The first thing that happens in CreateMap is setting `trajectory_id` so I am not too worried about using 0. Also I like setting an initialized value so its not using a random number if something bad happens

Reply:
In C++, writing `int trajectory_id;` and `int trajectory_id = 0;` is the same, since 0 is the default value for `int`. I'm also not worried about failing to set this value, since we set it immediately below.